Tutorial.md

```diff
@@ -0,0 +1,66 @@
+
+> **NOTE:** **Additional MSON Resources**
+>
+> + [MSON Specification](https://github.com/apiaryio/mson/blob/master/MSON%20Specification.md)
+> + [Examples](https://github.com/apiaryio/mson#notational-conventions)
+
+## Data structures in MSON
+
+To get started, we're going to create a data structure in MSON to represent a Person.
+
+A data structure to store a Person could be represented in MSON using the following:
+
+```apib
+# Person (object)
+
++ name
+```
+
+Here we've declared our own named type called "Person". This is indicated by using a heading. This is one or more `#` symbols followed by a string representing the name of our data structure, after the name we have declared in round brackets the type of data structure that a person is, in this case, it is an object.
+
+An object is constructed with properties, and in this case we've listed them below the heading in a list. Lists are created by preceding list items with either a `+`, `*` or `-`.
+
+In our "Person" data structure, we havbe a single property called `name`. Similary to our Person definition, we can declare the type that our property is, with round brackets. For example, we can delcare that our name is a string using `+ name (string)`. By default all types are string's unless specified to another type, as in our example we have ommitted the string type definition.
+
+We can attach a description for our name property by following the type definition with a dash (`-`) followed by the description.
+
+```apib
++ name (string) - The Person's name
+```
+
+We may also specify a sample value for the person's name by preceding the property name by a colon (`:`) and then the value.
+
+```apib
++ name: Kyle (string) - The Person's name
+```
+
+**NOTE**: *It's important to note, that if a sample value contains reserved characters such as `:`, `(`,`)`, `<`, `>`, `{`, `}`, `[`, `]`, `_`, `*`, `-`, `+`, `` ` `` then the sample value must be wrapped in a code-block using backticks:
+
+```apib
++ name: `Spencer-Churchill` (string) - The Person's name
+```
+
+So far we have used the object and string types, MSON includes a total of 6 base types. Three primitive types such as `boolean`, `string` and `number`. There are also three structure types such as `array`, `enum` and `object`. Let's use some of these new types to create another data structure.
+
+```apib
+# Company (object)
+
++ name: Apiary
++ founder (Person)
++ founded: 2011 (number) - The year in which the company was founded
+```
+
+### Inheritence
+
+When declaring a named type, you may also inherit from another type. For example, we could declare a data structure called "Administrator" which inherites from the "Person" type.
+
+```apib
+# Administrator (Person)
+
++ role (string) - The administrators role
+```
+
+When inheriting from another data structure, all of the parents properties will be inherited. For example, our "Administrator" structure will contain the "name" property from our previous definition.
```

Comment:
This should probably use `Administrator` instead of "Administrator", and the same goes for the `name` property.